prisma/schema.prisma

```diff
@@ -48,15 +55,22 @@ model User {
 }
 
 model Tournament {
-  id                    String   @id @unique @default(uuid())
-  host                  User     @relation(fields: [userId], references: [id])
-  hostingDetail         HostingDetails?
-  gamingDetail          GamingDetails?
+  id                    String         @id @unique @default(uuid())
+  host                  User           @relation(fields: [userId], references: [id])
+  hostingDetail         HostingDetails? @relation(fields: [hostedTournamentId], references: [id])
+  hostedTournamentId    String?
```

Comment:
@adityabhagat007 IG `hostingDetailId` maybe a good name instead of `hostedTournamentId`.